src/Speck.js

```diff
@@ -134,13 +134,32 @@ class Speck {
     return JSON.parse(JSON.stringify(rawData));
   }
 
+  toJSONContext(context) {
```

Comment:
@guisouza you got a strong point about `getContext`. If we choose this approach, we will need to review some points of Speck that need much more work than only create a JSON based in a context.

For now, we could use an api like `toJSONByContext` (I think that the main idea here is "Transform this entity in a JSON based on the fields declared on the x context") or even keep the `toJSON` method with an optional param to transform using a context, like: `user.toJSON({ context: 'astronaut' })`.